gui/pregnancy.lua

```diff
@@ -26,14 +40,14 @@ function PregnancyGui:init()
                 },
                 widgets.HotkeyLabel{
                     frame={l=0},
-                    label="Select Mother",
+                    label="Set mother to selected unit",
```

Comment:
you could also set `enabled=function() local unit = dfhack.gui.getSelectedUnit(true) return unit and unit.sex == df.pronoun_type.she end,`

Reply:
and similarly for the father HotkeyLabel